lib/iris/tests/unit/analysis/area_weighted/test_AreaWeightedRegridder.py

```diff
@@ -30,42 +31,53 @@ def cube(self, x, y):
         lon = DimCoord(x, "longitude", units="degrees")
         cube.add_dim_coord(lat, 0)
         cube.add_dim_coord(lon, 1)
+        cube.coord("latitude").guess_bounds()
+        cube.coord("longitude").guess_bounds()
         return cube
 
     def grids(self):
         src = self.cube(np.linspace(20, 30, 3), np.linspace(10, 25, 4))
-        target = self.cube(np.linspace(6, 18, 8), np.linspace(11, 22, 9))
+        target = self.cube(np.linspace(22, 28, 8), np.linspace(11, 22, 9))
         return src, target
```

Comment:
Alter this so that we are passing data through the tests. It is ok to do this because all the tests already using it abort with an error and are independent of the data used.